text_summarization/text_summarization.csproj

```diff
@@ -1,12 +1,24 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
     <OutputType>Exe</OutputType>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <UserSecretsId>4af7e2fb-2a2c-40e3-8beb-3ea5e81fc6c6</UserSecretsId>
+    <DefaultLocalSettingsJson>
```

Comment:
This creeps me out a little. JSON-inside-of-XML? It seems like we should be able to use `IConfiguration.GetValue<T>("")` and avoid the requirement of the JSON, but I get it may be a "functions Gestalt" to have the file there.